### ISSUE TITLE
Node title name (category name) not taken from storeview?

### ISSUE THREAD

**Issue by magex55** [SnowdogApps/magento2-menu#327]
I am having multiple storeviews in my store, representing languages. A menu node (category name) is called "Shoes" for example and in the german store view its called "Schuhe" ... but no matter what storeview I choose when creating the menu, it stays "Shoes". Thats just an example.

Am I doing this wrong, or does it not work? How is it possible to show the real category name which matches the store view?

Thanks

**Comment by andrzejborkowski** [comment]
Hi @magex55 
I was trying to replicate this issue on our menu stage (m2.4.5 + 2.26.0 menu version) and I can see correct submenu node name on frontend when switching store views:
![Screenshot 2024-05-22 at 14 08 49](https://github.com/SnowdogApps/magento2-menu/assets/73829518/6f973fda-b320-45b2-9710-eb46c7a76c92)

Did you clear cache after updating node name?

Regards,
Andrzej

**Comment by magex55** [comment]
I cleared the cache, yes, here is exactly what I did:

1. Created a menu and set my french storeview for it
2. Clicked the "import from categories button", which created all the nodes matching my category structure

Now I looked at the nodes in admin first and they did not have the french category names. Now I switched to the french store view on frontend, which made the store appear in french, but the menu nodes did not switch to french.

Maybe its a misunderstanding or something. Does the extension take the node name from the Magento category name according to the selected store view?

**Comment by magex55** [comment]
Yes, I think we are talking about the same thing now :-). I need to go live soon, do you think it will be solved anytime soon? Thanks so much!

**Comment by andrzejborkowski** [comment]
I can't provide a deadline at this time, so I suggest manually updating the category node names for go live. Thanks!

**Comment by ebaraut** [comment]
Any update on this issue? Thanks!

**Comment by andrzejborkowski** [comment]
@ebaraut This has already been released in version 2.28.0, please see the [release notes](https://github.com/SnowdogApps/magento2-menu/blob/develop/CHANGELOG.md#2280---2025-03-25).
Apologies for not notifying you about the release earlier.

**Comment by ebaraut** [comment]
Thanks, but is it possible to choose a store view as in "Import from categories"? I think the name is always in the same language.

**Comment by andrzejborkowski** [comment]
In version **2.28.0** of the Menu module, we added a **Store View dropdown** to the **Import from Categories** page. You can now select the appropriate store view from which to import category names. See the attached screenshot:

<img width="998" alt="Image" src="https://github.com/user-attachments/assets/f90df167-cc6a-4dc1-82a9-eef814b88f7a" />
